.github/workflows/reusable-build-upload.yaml

```diff
@@ -78,7 +78,7 @@ jobs:
           export PACKAGE_VERSION='v${{ inputs.package-version }}'
 
           export BASE_ARTIFACTS_DIR="./target/wasm32-unknown-unknown/release"
-          export ARTIFACT_NAME="axelar-cgp-soroban-wasm-${PACKAGE_NAME}-${PACKAGE_VERSION}"
+          export ARTIFACT_NAME="axelar-cgp-stellar-wasm-${PACKAGE_NAME}-${PACKAGE_VERSION}"
```

Comment:
where is there such a prefix? seems unnecessary verbose, talk to Ayoub about removing it